packages/nodes-base/nodes/Google/Firebase/CloudFirestore/GenericFunctions.ts

```diff
@@ -58,6 +60,7 @@ export async function googleApiRequestAllItems(this: IExecuteFunctions | ILoadOp
 	return returnData;
 }
 
+const isValidDate = (str: string) => moment(str, 'YYYY-MM-DD HH:mm:ss Z', true).isValid();
```

Comment:
Maybe adding **ISO_8601** format can make it easy for users to provide an ISO datetime as input.
```js
 const isValidDate = (str: string) => moment(str, ['YYYY-MM-DD HH:mm:ss Z', moment.ISO_8601], true).isValid();

```